packages/sources/src/styling/variables/zindex.scss

```diff
@@ -1,5 +1,31 @@
-$zindex-navbar: 1000;
-$zindex-dropdown: 1000;
-$zindex-popover: 1060;
-$zindex-tooltip: 1070;
-$zindex-navbar-fixed: 1030;
+// Selectors
+$zx-viewer-grid-sizing-buttons:     -1;
+$zx-viewer-grid-map-panel:          100;
+
+$zx-selectable:                     100;
+
+$zx-datepicker-base:                100;
+$zx-datepicker-dialog:              210;
+
+$zx-description-open:               100;
+
+$zx-searchbar-icon:                 101;
+
+$zx-form-control-feedback:          101;
+$zx-input-group:                    101;
+
+$zx-leaflet:                        110;
+
+// Dropdowns
```

Comment:
Soms wel comments, soms niet. Wat wordt nu de lijn? Ik zou zeggen ofwel haal alles weg, ofwel voorzie iedere variabele (zoveel zijn het er niet) van een toelichting over het hoe en wat.

Reply:
Deze kwamen voort uit het overzicht van Michiel, maar de lijst is nu zo ingekrompen (en onder verdeeld) dat dit nut ook een beetje verdwenen is.

Reply:
eens.

Reply:
eens.